ios/FluentUI.Demo/FluentUI.Demo/ReadmeViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+
+class ReadmeViewController: UIViewController {
+
+    let scrollView = UIScrollView()
+    let contentView = UIView()
+
+    let readme: String?
+
+    let readmeLabel: Label = {
+        let label = Label()
+        label.numberOfLines = 0
+        label.sizeToFit()
+        label.textColor = UIColor(dynamicColor: DynamicColor(light: ColorValue(0x000000),
+                                                             dark: ColorValue(0xFFFFFF)))
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupScrollViewConstraints()
+        setupReadmeLabelConstraints()
+        setReadmeLabelText()
+    }
+
+    private func setupReadmeLabelConstraints() {
+        contentView.addSubview(readmeLabel)
+        readmeLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor).isActive = true
+        readmeLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: Constants.topPadding).isActive = true
+        readmeLabel.widthAnchor.constraint(equalTo: contentView.widthAnchor, multiplier: Constants.widthMultiplier).isActive = true
+        readmeLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor).isActive = true
+    }
+
+    private func setupScrollViewConstraints() {
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(scrollView)
+        scrollView.addSubview(contentView)
+
+        scrollView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
```

Comment:
if you have bunch of constarints to activate, it is generally better to add it in a array and activate at once for better performance https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526955-activateconstraints#:~:text=Typically%2C%20using%20this%20method%20is%20more%20efficient%20than%20activating%20each%20constraint%20individually.